tests/dummy_hardware/platform.py

```diff
@@ -0,0 +1,16 @@
+"""Dummy platform to test loading via ``Hardware`` object."""
+
+from pathlib import Path
+
+from qibolab import Hardware, initialize_parameters
+from qibolab._core.dummy.platform import create_dummy_hardware
+from qibolab._core.platform.platform import PARAMETERS
+
+
+def create() -> Hardware:
+    hardware = create_dummy_hardware()
+    parameters = initialize_parameters(hardware=hardware)
+    (Path(__file__).parent / PARAMETERS).write_text(
+        parameters.model_dump_json(indent=4)
+    )
```

Comment:
Instead of making this part of the platform itself, I'd turn it into a fixture, such that the parameters will also get erased after the test is completed.

Reply:
The whole function will become just an alias of `create_dummy_hardware`, which is not bad, since it will have its own path, as an actual user-specified platform.

Reply:
fa1b22e26d91b4d341220c3a71c5c3020d87128b